Dockerfile

```diff
@@ -392,6 +368,22 @@ RUN boot -u
 
 USER root
 
+ADD lib/dependencies /php/dependencies/
+ADD lib/php/5.6 /php/5.6
+ADD lib/php/7.2 /php/7.2
+ADD lib/php/7.4 /php/7.4
```

Comment:
does this explicitly need `ADD` or could you use `COPY` instead? i think `COPY` is generally preferrable.